src/ejabberd_s2s_out.erl

```diff
@@ -998,9 +998,7 @@ get_addr_port(Server) ->
 
 -spec srv_lookup(jid:server()) -> {'error', atom()} | {'ok', inet:hostent()}.
 srv_lookup(Server) ->
-    Options = mongoose_config:get_opt(s2s_dns_options, []),
-    TimeoutMs = timer:seconds(proplists:get_value(timeout, Options, 10)),
```

Comment:
With `timer:seconds` here, it seems the DNS timeout was already 10 seconds, not milliseconds, so at least it wasn't so wrong. But the different time units in the config were probably confusing.

Reply:
I will fix this tomorrow. One timeout unit in the docs was wrong, I will think if I need to change the unit or the docs then.